SpatialConvolutionLocal.lua

```diff
@@ -0,0 +1,166 @@
+local SpatialConvolutionLocal, parent = torch.class('nn.SpatialConvolutionLocal', 'nn.Module')
+
+function SpatialConvolutionLocal:__init(nInputPlane, nOutputPlane, iW, iH ,kW, kH, dW, dH, padW, padH)
+   parent.__init(self)
+
+   dW = dW or 1
+   dH = dH or 1
+
+   self.nInputPlane = nInputPlane
+   self.nOutputPlane = nOutputPlane
+   self.kW = kW
+   self.kH = kH
+   self.iW = iW
+   self.iH = iH
+
+   self.dW = dW
+   self.dH = dH
+   self.padW = padW or 0
+   self.padH = padH or self.padW
+   self.oW = (self.padW * 2 + iW - self.kW) / self.dW + 1
```

Comment:
This needs to be fixed by adding a `floor`.
If not, you can have wrong results when viewing the weights, as in https://github.com/torch/nn/pull/546/files#diff-be444d1c4584d74f86d51ed473de5751R71